zap/src/main/java/org/zaproxy/zap/model/Tech.java

```diff
@@ -195,4 +206,68 @@ public int compareTo(Tech o) {
 
         return this.toString().compareTo(o.toString());
     }
+
+    /**
+     * Returns all techs
+     *
+     * @return all techs
+     */
+    public static Tech[] getAll() {
```

Comment:
- Should return a collection, rather than create a new array each time.
 - Should have a since tag.

Reply:
Hope v 2.10.0 works for @since